alerter.py

```diff
@@ -2,10 +2,10 @@
 
 def network_alert_stub(celcius):
     print(f'ALERT: Temperature is {celcius} celcius')
-    # Return 200 for ok
-    # Return 500 for not-ok
-    # stub always succeeds and returns 200
-    return 200
+    if celcius >= 100:
+        return 200
+    else:
+        return 500
 
 def alert_in_celcius(farenheit):
     celcius = (farenheit - 32) * 5 / 9

```

Comment:
Production code is still aware of a stub. This should be abstracted by the use of a function pointer. i.e. in test envt use network stub function and in production envt use the actual network component function.